app/models/sharing_strategies/work_package_strategy.rb

```diff
@@ -57,7 +57,7 @@ def viewable?
     # portal outlet when rendering, causing bugs. Until the work package share dialog is refactored to be an
     # async-dialog, we must disable hover cards for it.
     def allow_hover_cards?
```

Comment:
Do we still need that method at all? All occurences I found set it to `true` and the whole reason for it is obselete, isn't it?

Reply:
True, I even sneakily removed the comment just now, as it is out of date. Will get rid of the entire method!